File: code-recipes/php/test/easyship/GetScheduledPackageRecipeTest.php
<?php

namespace Test\easyship;

use Src\easyship\GetScheduledPackageRecipe;
use Test\util\RecipeTestCase;

class GetScheduledPackageRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new GetScheduledPackageRecipe();
        $this->responses = [
            "easyship-getScheduledPackage"
        ];
    }
}

File: code-recipes/java/src/test/java/easyship/GetScheduledPackageRecipeTest.java
package easyship;

import util.RecipeTest;

import java.util.List;

public class GetScheduledPackageRecipeTest extends RecipeTest {

    protected GetScheduledPackageRecipeTest() {
        super(
                new GetScheduledPackageRecipe(),
                List.of(
                    "easyship-getScheduledPackage"
                )
        );
    }
}
